### DISCUSSION TITLE
[Errno 99] error while attempting to bind on address：cannot assign requested address

### DISCUSSION THREAD

**Discussion by kli017** [fastapi/fastapi#6405] [Questions]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [ ] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [ ] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [ ] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [ ] I commit to help with one of those options 👆

### Example Code

```python
from fastapi import FastAPI, File, UploadFile

# API for single wav file uploading
@app.post("/uploadfile/")
def create_upload_file(upload_file: UploadFile = File(...)):
    uuid = uid.uuid4()
    file_folder = f"files/{uuid}"
    file_location = f"files/{uuid}/{upload_file.filename}"
    if not os.path.exists(file_folder):
        os.makedirs(file_folder)
    with open(file_location, "wb+") as file_object:
        file_object.write(upload_file.file.read())
    
    logger.info("file {} saved!".format(upload_file.filename))
    return {"info": f"file '{upload_file.filename}' saved at '{file_folder}'",
            "uuid": uuid}

if __name__ == '__main__':
    uvicorn.run(app=app, host="172.16.20.105", port=8090)
```

### Description

I tested my code at local and the api service runs good. For some reason I need to set the service in a docker for usage. So I created a docker image by Dockerfile. However when I run the docker I got the following error:
INFO:     Started server process [7]
INFO:     Waiting for application startup.
INFO:     Application startup complete.
ERROR:    [Errno 99] error while attempting to bind on address ('172.16.20.105', 8090): cannot assign requested address
INFO:     Waiting for application shutdown.
INFO:     Application shutdown complete.

I checked the port and for sure it is not token by other process. The service could run on the same address while start in the local. 

### Operating System

Linux

### FastAPI Version

0.75.1

### Python Version

3.8.0

**Comment by tiangolo** [comment]
I assume you solved your problem, thanks for coming back to close the issue!

> Sorry for the long delay! 🙈  I wanted to personally address each issue and they piled up through time, but now I'm checking each one in order.

**Comment by DVKsensibelle** [comment]
How you solved this issue?

**Reply by YuriiMotov** [reply]
Check that you don't have other copy of this application already running.
Sometimes it happens that my server is stopped, but I still have this error. In this case, a system reboot should help.
If none of above helped you, it's probably another service that works on this port. Try using another port.

**Reply by DVKsensibelle** [reply]
Thanks a lot!

Problem was really with opened application in jupyter, while I was trying to raise docker.
